README.md

```diff
@@ -217,4 +222,55 @@ To add file watcher for transpiling the scss files
 
 ```
 npm run dev-webplugin-scss
-```
\ No newline at end of file
+```
```

Comment:
The below Readme is very extensive but it could be simplified to something like this:

To generate a development package use:

------------
```
npm run genereate-webplugin-*-dev
```

Where the '*' can be replaced with one of the following browser names:
- chrome (also used for Opera)
- firefox
- safari
- edge
------------

The same can be applied to the production build. Reiterating the available browsers should not be needed in this case.